.github/workflows/build.yml

```diff
@@ -11,6 +11,7 @@ on:
         options:
           - Beta
           - Production
+          - Simulator
```

Comment:
Hm. I like that idea, and it makes sense, in case we need to run the Beta app on the simulator. I'll chew on the name for this (Physical Device vs. Simulator), and split it out. That'll impact the conditionals, but not in a bad way I don't think

Reply:
I ultimately decided on "build mode" mostly because that's what I had in the build script already for that

Reply:
Ship it 🚀